src/features/profile/components/SectionRow.tsx

```diff
@@ -30,8 +32,8 @@ export function SectionRow(props: SectionRowProps) {
     <TouchableOpacity
       activeOpacity={props.onPress ? ACTIVE_OPACITY : 1}
       onPress={props.onPress}
-      testID="section-row-touchable">
-      <Container>
+      testID={props.testID ? props.testID : 'section-row-touchable'}>
+      <View style={[styles.container, props.style]}>
```

Comment:
Il y a peut-être un truc que je ne comprends pas mais pourquoi pas :
```suggestion
      <Container style={props.style}>
``` 
comment dans `ProfileSection` ?